docs/xdsl-introduction.ipynb

```diff
@@ -0,0 +1,937 @@
+{
```

Comment:
<p>for analyzing the code..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/398/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>updated</p>